cli/command/container/tty.go

```diff
@@ -54,8 +54,8 @@ func initTtySize(ctx context.Context, cli command.Cli, id string, isExec bool, r
 	if err := rttyFunc(ctx, cli, id, isExec); err != nil {
 		go func() {
 			var err error
-			for retry := 0; retry < 5; retry++ {
-				time.Sleep(10 * time.Millisecond)
+			for retry := 0; retry < 10; retry++ {
+				time.Sleep(100 * time.Millisecond)
```

Comment:
Wondering if we should consider making the delay incremental;

- retry after 10 milliseconds ("happy path" for when the daemon wasn't able to handle it immediately)
- if failed; add 10 milliseconds (so, retry after 20 milliseconds)
- if failed; add 10 milliseconds (so, retry after 30)
- and so on, until we reach 10 attempts (which would be using 100 milliseconds) 

If incrementing by 10 is not "aggressive" enough, perhaps we could make it exponential (but with a "cap", which could be 100ms);

10 -> 20 -> 40 -> 80 -> 100 -> 100 -> 100


Doing so would allow the resize to be handled "faster" (in the happy case), while still _allowing_ it to take longer (if the daemon happens to be under load)

WDYT?

Reply:
Done!